test/suites/basic.sh

```diff
@@ -672,6 +672,26 @@ test_basic_usage() {
   lxc profile delete foo
   lxc delete -f c1
 
+  # Test assigning a profile through a YAML file to an instance.
+  poolName=$(lxc profile device get default root pool)
+  lxc profile create foo < <(cat <<EOF
+config:
+  limits.cpu: 2
+  limits.memory: 1024MiB
+description: Test profile
+devices:
+  root:
+    path: /
+    pool: ${poolName}
+    type: disk
+EOF
+)
+  lxc init testimage c1 --profile foo
+  [ "$(lxc config show c1 --expanded | yq e '.config."limits.cpu"' -)" = "2" ]
```

Comment:
We dont need another tool, we can should just use `lxc config get c1 limits.cpu --expanded`

Reply:
fixed